lib/Dialect/FIRRTL/Transforms/GrandCentralTaps.cpp

```diff
@@ -298,16 +298,16 @@ class GrandCentralTapsPass : public GrandCentralTapsBase<GrandCentralTapsPass> {
   void gatherTap(Annotation anno, Port port) {
     auto key = getKey(anno);
     annos.insert({key, anno});
-    auto it = tappedPorts.insert({key, port});
-    assert(it.second && "ambiguous tap annotation");
+    tappedPorts.insert({key, port});
+    assert(tappedPorts.find(key)->second && "ambiguous tap annotation");
```

Comment:
I don't think this will work properly, try this instead:
```c++
assert(!tappedPorts.count(key) && "ambiguous tap annotation");
tappedPorts.insert({key, port});
```
or you can use `opNames.find(op) == opNames.end()` in the assert as in `SCFToCalyx.cpp`.  The same change is needed below.